system/HTTP/RequestTrait.php

```diff
@@ -240,14 +240,14 @@ public function setGlobal(string $method, $value)
 	 *
 	 * http://php.net/manual/en/filter.filters.sanitize.php
 	 *
-	 * @param string            $method Input filter constant
-	 * @param string|array|null $index
-	 * @param integer|null      $filter Filter constant
-	 * @param mixed             $flags
+	 * @param string             $method Input filter constant
+	 * @param string|array|null  $index
+	 * @param integer|null       $filter Filter constant
+	 * @param array|integer|null $flags  Options
 	 *
 	 * @return mixed
 	 */
-	public function fetchGlobal($method, $index = null, $filter = null, $flags = null)
+	public function fetchGlobal(string $method, $index = null, ?int $filter = null, $flags = null)
```

Comment:
```suggestion
	public function fetchGlobal(string $method, $index = null, ?int $filter, $flags = null)
```

Reply:
1. A nullable type assumes NULL, but does not define a default.
2. Violating php 8 compatibility